### ISSUE TITLE
About train

### ISSUE THREAD

**Issue by xuliu-cyber** [AhmedBourouis/Scene-Sketch-Segmentation#1]
Hi, Thanks for your excellent work! I tried to reproduce the results following the training tutorial in readme.txt, but I got the evaluation results lower than the paper. Could you tell me why?

**Comment by AhmedBourouis** [comment]
Hi @xuliu-cyber, can you please share more details about the training loss over epochs and the results you got?

**Comment by xuliu-cyber** [comment]
Thanks for your reply. The result I got is shown in the bottom of picture. And I didn't generate log files during training. The directory of the CUB is empty.
![QQ截图20240529182038](https://github.com/AhmedBourouis/Scene-Sketch-Segmentation/assets/61044387/c24e929c-543b-436e-b77c-1329af2a00ab)

**Comment by AhmedBourouis** [comment]
Thanks @xuliu-cyber. I would suggest setting augment to "True" when loading the training data and refer you to section S4.2 in the paper supplementary for more details on how the best model checkpoint was selected.
